### PR TITLE
Able to delete in group for delete feature

### PR DESCRIPTION
**Pull request by cheahTJ**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -21,29 +25,69 @@ public class DeleteCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the NusId used in the displayed person list.\n"
             + "Parameters: NusId (8 digits long, starting with an 'E'). \n"
-            + "Example: " + COMMAND_WORD + " E0123456";
+            + "Example: " + COMMAND_WORD + " id/E0123456\n\n"
```
Comment (hjuntan) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:28]: Not sure if this is intentional but there are a double \n here

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ public void parseCommand_clear() throws Exception {
     public void parseCommand_delete() throws Exception {
         NusId testNusId = new NusId("E1234567");
         DeleteCommand command = (DeleteCommand) parser.parseCommand(
-                DeleteCommand.COMMAND_WORD + " " + testNusId);
+                DeleteCommand.COMMAND_WORD + " id/" + testNusId);
```
Comment (hjuntan) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:54]: I think we can use the ParserUtil prefix instead of adding it ourselves

**Review by hjuntan — COMMENTED** [review]
Overall looks good to me. The stuff mentioned are things that can be improved on but it does not require immediate attention.

**Comment by hjuntan** [comment]
Closes #51

File: src/main/java/seedu/address/logic/parser/GroupCommandParser.java [hunk 3]
```diff
@@ -38,6 +38,12 @@ public GroupCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, GroupCommand.MESSAGE_USAGE));
         }
 
+        if (arePrefixesPresent(argMultimap, PREFIX_NUSID)
+                && !arePrefixesPresent(argMultimap, PREFIX_GROUP)
+                && !arePrefixesPresent(argMultimap, PREFIX_TAG)) {
```
Comment (RingoftheKing) [inline src/main/java/seedu/address/logic/parser/GroupCommandParser.java:43]: Maybe we really need to create a common method in ParserUtils that accepts an array of args

**Pull request merged**